stream.go

```diff
@@ -132,12 +132,10 @@ func (s *Stream) WriteSysExBytes(when Timestamp, msg []byte) error {
 // the output stream. The string must only consist of hex digits (0-9A-F) and optional spaces. This
 // function is case-insenstive.
 func (s *Stream) WriteSysEx(when Timestamp, msg string) error {
-	buf, err := hex.DecodeString(strings.Replace(msg, " ", "", -1))
```

Comment:
Forgot all about that, that was from a fork by mkb218. Sending one voice over sysex to the TX7 is a 163 byte package, hex.DecodeString doesn't like odd lengths. 

```
encoding/hex: invalid byte: U+00F0 'ð'
encoding/hex: odd length hex string
```
I didn't merge it in the git way (apologies), but I am happy to revert it and merge it in properly. I just gave credit in the commit message before I was thinking about contributing back.